### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by sherl0cks**
This is an initial commit that:

- adds a `mvn` pipeline
- adds a `npm` pipeline
- updates docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-# pipeline-library
-A repository of Jenkins pipeline files we can reference from elsewhere
+# What This Repo Is
+
+A repository of application centric [Jenkins pipeline files](https://jenkins.io/doc/book/pipeline/) that be references or reused for appropriate applications.
```
Comment (etsauer) [inline README.md]: `s/that be references/that can be referenced/`

Comment (sherl0cks) [reply README.md]: Fixed

**Comment by sherl0cks** [comment]
@sabre1041 wanna take a look?

**Comment by sabre1041** [comment]
LGTM

**Pull request merged**